configure.ac

```diff
@@ -432,8 +432,18 @@ OMRCFG_DEFINE_FLAG([OMR_ENV_DATA64],[],
 	)]
 )
 
-OMRCFG_DEFINE_FLAG_OFF([OMR_GC_COMPRESSED_POINTERS])
-OMRCFG_DEFINE_FLAG_OFF([OMR_GC_FULL_POINTERS])
+AC_ARG_VAR([OMR_GC_POINTERS], [The mode of pointers.  Must be \"full\", \"compressed\", or \"mixed\"])
+: ${OMR_GC_POINTERS="full"}
+
+OMRCFG_DEFINE_FLAG([OMR_GC_COMPRESSED_POINTERS],[],
+	[AS_IF([[test "x$OMR_GC_POINTERS" = "mixed"] || [test "x$OMR_GC_POINTERS" = "compressed"]],
+		[OMRCFG_ENABLE_FLAG([OMR_GC_COMPRESSED_POINTERS])],
+		[OMRCFG_DISABLE_FLAG([OMR_GC_COMPRESSED_POINTERS])])])
```

Comment:
I actually think we don't want to use `OMRCFG_DEFINE_FLAG`, because `OMR_GC_COMPRESSED_POINTERS ` and `OMR_GC_FULL_POINTERS` are not a user-facing options. It's better if we just use `AC_SUBST`.

Reply:
OMRCFG_ENABLE_FLAG does not add it as a command line options, it substitutes the value into the configuration headerfile and makefile